apps/agentic-chat/src/components/ui/StatusText.tsx

```
@@ -0,0 +1,40 @@
+function StatusTextComponent({ children }: { children: React.ReactNode }) {
+  return <div className="text-base text-gray-600">{children}</div>
+}
+
+StatusTextComponent.Loading = function Loading({ children }: { children: React.ReactNode }) {
+  return <div className="text-base text-blue-600">{children}</div>
+}
+
+StatusTextComponent.Error = function Error({ children }: { children: React.ReactNode }) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)